✨ feat: added new icons for tools and softwareIDEs

eudanielhenrique opened the pull request:
Added to file "github-widgetbox/src/data/software-ides.ts"

Windows operating system icon and Visual Studio Code

eudanielhenrique committed:
✨feat: new icon to system windows

eudanielhenrique committed:
✨ feat: new icon to system vscode

eudanielhenrique committed:
💥 fix: reverting changes

eudanielhenrique committed:
✨ feat: new icon to system windows and vscode

eudanielhenrique committed:
📚 docs: update do README

add themes section, with 6 examples

eudanielhenrique committed:
📚 docs: create THEMES.MD

showing some themes, I haven't put them all yet

Jurredr commented on src/data/software-ides.ts:
```diff
@@ -31,6 +31,73 @@ const softwareIDEs: IconData[] = [
     },
+
+    {
+        name: ['vscode', 'Visual Studio Code'],
```
Small nitpick; change capitalisation to "VSCode" as the first string is the exact way it would show up with `includeNames=true`. Also, spaces in the name will not work as they cannot be typed in the URL anyways.

Jurredr commented on README.md:
```diff
@@ -70,12 +70,36 @@ Show off your profile with some interesting statistics. Perfect for profile READ
+<br/><br/>
+### Themes
+
+It can vary in the different themes that we have ready for you, being able to check all the available themes in  [themes.md](https://github.com/Jurredr/github-widgetbox/blob/master/THEMES.md)
```
Add a link to the themes TypeScript file as well, as not all themes are displayed in themes.md yet. Also maybe add a short line saying that people can make their own themes by following the format there and making a PR?

eudanielhenrique replied:
Interesting the idea of people creating their own themes, I'll put them to follow to the .ts file as well.

Jurredr reviewed (commented):
Left 2 comments, overall looks good, thanks!

eudanielhenrique committed:
💥 fix: reverting changes

eudanielhenrique committed:
📚 docs: update README

Adding the user to where the themes are and saying that they can create their own themes.

eudanielhenrique reviewed (commented):
Corrections made, does it agree with what I had imagined?

Jurredr commented:
Looks good, merged!
